mapping/sustainability.yaml

```
@@ -1293,12 +1293,12 @@
 
         Publish a summary of the labor obligations:
 
-        1. For each labor obligation in the contract, add a code from the laborObligations codelist to the`.summary.social.laborObligations.obligations` array.
+        1. For each labor obligation in the contract, add a code from the [laborObligations](https://standard.open-contracting.org/infrastructure/%7B%7Bversion%7D%7D/%7B%7Blang%7D%7D/reference/codelists/#laborobligations) codelist to the`.summary.social.laborObligations.obligations` array.
```

The mappings are rendered as Markdown on `sustainability.md` so we should use relative links, same as on other Markdown pages.

```suggestion
        1. For each labor obligation in the contract, add a code from the [laborObligations](../../reference/codelists.md#laborobligations) codelist to the`.summary.social.laborObligations.obligations` array.
```

---

ah, the one example I could find in the reviewed PR's at this point as them as full urls, so you'll find a few of these in various PR's I've done up to now